src/hooks/useSignErrorHandler.test.ts

```diff
@@ -0,0 +1,196 @@
+import { act } from '@testing-library/react';
+
+import { useSignErrorHandler } from './useSignErrorHandler';
+import useToastMsg from './useToastMsg';
+import type { VCSignError } from './useVerifiableCredential';
+import { VCSignErrorType } from './useVerifiableCredential';
+import { renderHook } from '../utils/test-utils';
+
+jest.mock('./useToastMsg');
+
+describe('useSignErrorHandler', () => {
+  let useToastMsgMock: jest.Mock;
+  let showErrorMsgMock: jest.Mock;
+
+  beforeEach(() => {
+    useToastMsgMock = useToastMsg as jest.Mock;
+    useToastMsgMock.mockClear();
+
+    showErrorMsgMock = jest.fn();
+
+    useToastMsgMock.mockReturnValue({
+      showErrorMsg: showErrorMsgMock,
+    });
+  });
+
+  it('should show error message when signError override messages is defined', async () => {
+    const signError: VCSignError = {
+      type: VCSignErrorType.SignError,
+    };
+    const errorMessages = {
+      SignError: {
+        title: 'Custom Sign Error Title',
+        description: 'Custom Sign Error Description',
+      },
+    };
+
+    await act(() =>
+      renderHook(() => useSignErrorHandler(signError, errorMessages)),
+    );
+
+    expect(showErrorMsgMock).toHaveBeenCalledWith({
+      title: 'Custom Sign Error Title',
+      description: 'Custom Sign Error Description',
+    });
+  });
+
+  it('should show error message when VerifyError override messages is defined', async () => {
+    const signError: VCSignError = {
+      type: VCSignErrorType.VerifyError,
+    };
+    const errorMessages = {
+      VerifyError: {
+        title: 'Custom Sign Error Title',
+        description: 'Custom Sign Error Description',
+      },
+    };
+
+    await act(() =>
+      renderHook(() => useSignErrorHandler(signError, errorMessages)),
+    );
+
+    expect(showErrorMsgMock).toHaveBeenCalledWith({
+      title: 'Custom Sign Error Title',
+      description: 'Custom Sign Error Description',
+    });
+  });
+  it('should show error message when VerifyFailed override messages is defined', async () => {
```

Comment:
according to metamask guideline
we dont use "should"

instead
it('shows error.....')

please update allow below

Reply:
change all `should` in this file to use `Shows` and `Not show`